README.md

```diff
@@ -55,42 +63,67 @@ run the entire CMR with no external dependencies from this Jar file and use
 that instance for local testing. The sections below contain instructions for
 running the CMR as a single process or as multiple processes.
 
+#### First Steps
+
+1. Ensure you have installed on your system the items listed above in the
+   "Prerequisites" section.
+1. Download the Oracle JDBC `.jar` files into `./oracle-libs/support` by
+   following instructions in `./oracle-lib/README.md`. (The CMR must have these
+   libraries to build but it does not depend on Oracle DB when running
+   locally. It uses a local in-memory database by default.)
+
+#### Using the `cmr` CLI Tool
+
+This project has its own tool that is used for everything from initial setup to
+running buids and tests on the CI/CD infrastructure. In order to use the tool
+as we do below, be sure to run the following from the top-level CMR directory:
+
+```
+export PATH=$PATH:`pwd`/bin
+source resources/shell/cmr-bash-autocomplete
+```
+
+If you use another system shell, we'll except a PR with auto-complete for it.
+
 #### Building and Running CMR Dev System in a REPL
 
-1. Install Oracle JDBC Jars into your local maven repository following
-   instructions in `oracle-lib/README.md`. The CMR must have these libraries to build but it does not depend on Oracle DB when running locally. It uses a
-   local in-memory database by default.
-2. `cd cmr`
-3. `./dev-system/support/setup_local_dev.sh`
-4. `cd dev-system`
-4. `lein repl`
-5. Once given a Clojure prompt, run `(reset)`
+1. `cmr install oracle-libs`
+1. `cmr setup profile` and then update the new `./dev-system/profiles.clj`
```

Comment:
😦  - can we do anything for an outsider here?

Reply:
I've just been following the party line on this one ... would LOVE to provide official guidance here!